src/test/unit/StrategyManagerUnit.t.sol

```diff
@@ -240,7 +427,7 @@ contract StrategyManagerUnitTests is Test, Utils {
         strategyManager.depositIntoStrategy(dummyStrat, dummyToken, amount);
     }
 
-    function testDepositIntoStrategyRevertsWhenReentering() public {
+    function test_Revert_WhenReentering() public {
```

Comment:
technically wouldn't we want an equivalent test for `depositIntoStrategyBySignature` as well?

Reply:
Makes sense, updated tree file with changes to reflect the code paths as opposed to the testing coverage.
Will go back now to add missing tests and coverage